packages/sso-express/src/controllers.ts

```diff
@@ -109,7 +109,9 @@ export const loginController =
       code_challenge: codeChallenge,
       code_challenge_method: "S256",
       redirect_uri: redirectUri,
+      ...(options.authorizationUrlParams && { ...options.authorizationUrlParams }),
```

Comment:
you don't need to verify the presence of `authorizationUrlParams` since it's defaulting to {}